v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -272,6 +272,10 @@ export const GraphDataConfigurationModel = DataConfigurationModel
       const yHasCategorical = attrTypes.left === "categorical"
       return (xHasCategorical && !yHasCategorical) || (!xHasCategorical && yHasCategorical)
     },
+    get leftBottomCategoricalAttrCount() {
+      const attrTypes = self.attrTypes
+      return (attrTypes.left === "categorical" ? 1 : 0) + (attrTypes.bottom === "categorical" ? 1 : 0)
```

Comment:
Shouldn't this be:
```suggestion
      return (isCategoricalAttributeType(attrTypes.left) ? 1 : 0) + (isCategoricalAttributeType(attrTypes.bottom) ? 1 : 0)
```
so that, for instance, `color` attributes are handled as well?